### PR TITLE
(major) Added max-len rule

### LINKED ISSUE #7: Restrict line length
Should we add the [`max-length`](http://eslint.org/docs/rules/max-len) rule, and if so, with what configuration?

### PR DESCRIPTION
**Pull request by MitMaro**
Requires a major bump

Closes #7

### CODE REVIEW

File: rules/stylistic-issues.js [hunk 1]
```diff
@@ -47,7 +47,14 @@ module.exports = {
 		'linebreak-style': 'error',
 		'lines-around-comment': 'off',
 		'max-depth': 'error',
-		'max-len': 'off',
+		'max-len': [
+			'error',
+			{
+				code: 120,
+				ignoreTrailingComments: true,
```
Comment (erinwu) [inline rules/stylistic-issues.js]: do we want to set it to true? If the trailing comments are over 120, we can just put it as a comment block above the code.

**Comment by erinwu** [comment]
left a note about ignoreTrailingComments setting. Otherwise, all good. Approved.

File: rules/stylistic-issues.js (hunk 1 above)
Comment (dvalentiate) [inline rules/stylistic-issues.js]: this way trailing comments will be encouraged because they will be easier. I like `ignoreTrailingComments: true`

**Comment by dvalentiate** [comment]
approved

**Pull request merged**